src/hardware/pros_motor.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include "constant/constants.h"
+#include "interface/encoder.h"
+#include "interface/motor.h"
+#include "pros.h"
+
+namespace hardware {
+
+enum class ProsMotorCartridge {
+  // equal to pros value to enable casting
+  kBlueCartridge = pros::E_MOTOR_GEAR_BLUE,   // 600 rpm
+  kGreenCartridge = pros::E_MOTOR_GEAR_GREEN  // 200 rpm
+};
+
+pros::motor_gearset_e_t ConvertProsMotorCartridge(ProsMotorCartridge);
```

Comment:
What is the thought about making a wrapper enumeration around the pros enumerations for each gear cartridge? I agree with adding it, but there is some cost/benefit here that I'm curious to hear your perspective on

Reply:
The main justification is basically "ooga booga, pros ugly, wrapper clean". For example, I hate how pros uses `E`, doesn't use an enum class, allows you to reference directly with no namespace (e.g. `MOTOR_GEAR_GREEN`), and has multiple aliases (e.g. `pros::E_MOTOR_GEARSET_36`, `pros::E_MOTOR_GEAR_RED`, and `pros::E_MOTOR_GEAR_100`). The aliases in particular are awful since it isn't clear what's correct or not for things like switch statements. Plus, this way the pros library theoretically never has to be imported outside of hardware.